### PR TITLE
Book component

### PR DESCRIPTION
**Pull request by oliferwolox**
## Summary

- Add book component.
- Add alias for principal folders.

## Screenshot

Screenshot_1586374388

## Trello Card

https://trello.com/c/OE3bE51G/10-componente-libro

### CODE REVIEW

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  plugins: [
+    "@babel/plugin-proposal-optional-chaining",
+    [
+      "module-resolver",
+      {
+        "root": ["./src"],
+        "alias": {
+          "~components": "./src/app/components",
```
Comment (IgnacioAbadie) [inline .babelrc.js:9]: we are using `@` instead of `~` but i let it to @marianozicavo

File: styles.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {StyleSheet} from 'react-native';
+import {COLORS} from './src/constants/colors';
```
Comment (IgnacioAbadie) [inline styles.js:2]: add spaces between braces `{ COLORS }` . Same apply for all imports

File: src/app/components/Book/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {string} from 'prop-types';
+import {Text, View, Image, ActivityIndicator} from 'react-native';
+
+import styles from './styles';
+
+function Book({bookImage, bookTitle, bookAuthor}) {
```
Comment (IgnacioAbadie) [inline src/app/components/Book/index.js:7]: `({ bookImage, bookTitle, bookAuthor })`  add speces between braces and object content.

**Comment by IgnacioAbadie** [comment]
Take care with the commit names 😆

File: src/app/components/Book/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <View style={styles.book}>
+      <Image
+        source={{uri: bookImage}}
+        style={styles.image}
+        PlaceholderContent={<ActivityIndicator />}
```
Comment (marianozicavo) [inline src/app/components/Book/index.js:13]: It seems like this is a prop  of react native elements, but we aren't using that here. The `Image` you are using doesn't have a PlaceholderContent prop

File: src/app/components/Book/styles.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    flexDirection: 'row',
+    margin: 20,
+    padding: 20,
+    shadowColor: COLORS.black,
+    shadowOffset: {width: 1, height: 2},
+    shadowOpacity: 0.9,
+    width: 350,
+  },
+  bookInfo: {
+    flexDirection: 'column',
```
Comment (marianozicavo) [inline src/app/components/Book/styles.js:19]: That's the default `flexDirection`, you don't need this style

**Pull request merged**